### ISSUE TITLE
Plugin Issue - No Response

### ISSUE THREAD

**Issue by jeylites** [kiwi-cam/homebridge-broadlink-rm#47]
@kiwi-cam 

Don't know what happened, I think there's an issue with the latest update. Homekit accessories showing up as no response. 

**My setup**
- RPI3 B,  Homebridge Image 
- Broadlink RM3 mini
- Pluging [homebridge-broadlink-rm-pro v4.2.9 (2020-08-11)](https://www.npmjs.com/package/homebridge-broadlink-rm-pro)

**Homebridge Logs**                    
   ```
 ┌────────────┐     
    │ 734-47-253 │     
    └────────────┘     
                       
[9/1/2020, 16:57:16] Homebridge is running on port 51155.
[9/1/2020, 16:57:16] [Broadlink RM] Air Conditioner Advanced getCurrentHeatingCoolingState: 0
[9/1/2020, 16:57:16] [Broadlink RM] Air Conditioner Advanced getTargetHeatingCoolingState: 0
[9/1/2020, 16:57:16] [Broadlink RM] TV On/Off Test getSwitchState: false
[9/1/2020, 16:57:17] [Broadlink RM] [INFO] Discovered Broadlink RM Mini (2737) at 192.168.1.114 
[9/1/2020, 16:57:17] [Broadlink RM] Air Conditioner Advanced monitorTemperature
[9/1/2020, 16:57:17] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/1/2020, 16:57:18] [Broadlink RM] 
[9/1/2020, 16:57:18] [Broadlink RM] **************************************************************************************************************
[9/1/2020, 16:57:18] [Broadlink RM] ** Welcome to version 4.2.9 of the Homebridge Broadlink RM Plugin!
[9/1/2020, 16:57:18] [Broadlink RM] ** 
[9/1/2020, 16:57:18] [Broadlink RM] ** Find out what's in the latest release here: https://github.com/kiwi-cam/homebridge-broadlink-rm/blob/master/CHANGELOG.md
[9/1/2020, 16:57:18] [Broadlink RM] ** 
[9/1/2020, 16:57:18] [Broadlink RM] ** If you like this plugin then please star it on GitHub or better yet
[9/1/2020, 16:57:18] [Broadlink RM] ** buy me a drink using Paypal https://paypal.me/kiwicamRM.
[9/1/2020, 16:57:18] [Broadlink RM] **
[9/1/2020, 16:57:18] [Broadlink RM] ** You can disable this message by adding "hideWelcomeMessage": true to the config (see config-sample.json).
[9/1/2020, 16:57:18] [Broadlink RM] **
[9/1/2020, 16:57:18] [Broadlink RM] **************************************************************************************************************
[9/1/2020, 16:57:18] [Broadlink RM] 
[9/1/2020, 16:57:19] [Broadlink RM] Air Conditioner Advanced getTargetHeatingCoolingState: 0
[9/1/2020, 16:57:19] [Broadlink RM] Air Conditioner Advanced getCurrentHeatingCoolingState: 0
[9/1/2020, 16:57:19] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/1/2020, 16:57:19] [Broadlink RM] Air Conditioner Advanced getTargetTemperature: undefined
[9/1/2020, 16:57:19] [Broadlink RM] TV On/Off Test getSwitchState: false
[9/1/2020, 16:57:19] [Broadlink RM] Scan Frequency getSwitchState: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Switch Test getSwitchState: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Fan Test getFanSpeed: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Fan Test getRotationDirection: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Fan Test getSwingMode: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Fan Test getSwitchState: undefined
[9/1/2020, 16:57:19] [Broadlink RM] Learn getSwitchState: undefined
[9/1/2020, 16:57:27] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/1/2020, 16:57:37] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/1/2020,
```

**Config**

```
{
    "bridge": {
        "name": "Homebridge ",
        "username": "",
        "port": 51155,
        "pin": "734-47-253"
    },
    "hosts": [
        {
            "address": "192.168.1.104",
            "mac": "",
            "isRFSupported": true
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "accessories": [
                {
                    "name": "Air Conditioner Advanced",
                    "type": "air-conditioner",
                    "allowResend": true,
                    "preventResendHex": true,
                    "ignoreTemperatureWhenOff": true,
                    "replaceAutoMode": "cool",
                    "autoSwitch": "A/C Auto Switch",
                    "hideWelcomeMessage": true,
                    "maxTemperature": 24,
                    "temperatureUpdateFrequency": 10,
                    "defaultCoolTemperature": 13,
                    "defaultHeatlTemperature": 24,
                    "data": {
                        "off": "26008a0093530b0d0b200b200d0c0b1f0c0c0c0d0b0d0b1f0c0c0c0c0c0c0b0d0b0d0b0d0d1e0d0c0b0c0c0c0c0d0b0d0b0d0b0c0c0c0c0c0b0d0b0d0b200d1e0d0c0c0c0c0c0c0c0c0c0c0c0c0c0b0d0b0e0a0d0d0c0b0c0c0d0b0c0c0d0b0c0c0c0c0c0c0c0c1f0b200b0d0b0d0d0c0c1e0c0c0b0d0b200d0c0b0c0c1f0c1f0c1f0c1f0c0c0c00029095000d050000000000000000000000000000",
                        "temperature10": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature11": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature12": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature13": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature14": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature15": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature16": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature17": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature18": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature19": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature20": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature21": {
                            "pseudo-mode": "cool",
                            "data": "HEX"
                        },
                        "temperature22": {
                            "pseudo-mode": "cool",
                            "data": "HEX "
                        },
                        "temperature23": {
                            "pseudo-mode": "cool",
                            "data": "HEX "
                        },
                        "temperature24": {
                            "pseudo-mode": "cool",
                            "data": "HEX "
                        }
                    }
                },
                {
                    "name": "Switch Test",
                    "type": "switch"
                },
                {
                    "name": "Fan Test",
                    "type": "fan",
                    "data": {
                        "on": "2600500000012...",
                        "off": "2600500000012...",
                        "swingToggle": "2600500000012...",
                        "fanSpeed10": "2600500000012...",
                        "fanSpeed20": "2600500000012...",
                        "fanSpeed30": "2600500000012...",
                        "fanSpeed40": "2600500000012...",
                        "fanSpeed50": "2600500000012...",
                        "fanSpeed60": "2600500000012...",
                        "fanSpeed70": "2600500000012...",
                        "fanSpeed80": "2600500000012...",
                        "fanSpeed90": "2600500000012...",
                        "fanSpeed100": "2600500000012..."
                    }
                },
                {
                    "name": "TV On/Off Test",
                    "type": "switch",
                    "data": {
                        "on": "HEX",
                        "off": "HEX"
                    }
                }
            ]
        }
    ],
    "accessories": []
}
```

**Comment by kiwi-cam** [comment]
Thanks for the information.  4.2.9 just added a device type (610f). Could you confirm if 4.2.8 works?

It looks like something is up with the devices it's finding. I'm assuming you're using an RM3 Pro, given you're sourcing the temperature from the RM device (well, trying to). It's finding the device `Broadlink RM Mini (2737) at 192.168.1.114`. RM Mini's don't include a thermometer so will not return a temperature. 

You also specify a host 192.168.1.104 but this device doesn't appear to be discovered on the network. Have you configured this with the cloud, or changed anything else on it recently? Maybe try resetting it and re join it to the Wifi (making sure to abort before adding it to a room). I believe you've also got the "hosts" section in the wrong place, it should be here:
```
"platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "hosts": [
                    {
                        "address": "192.168.1.104",
                        "mac": "",
                        "isRFSupported": true
                    }
                ],
            "accessories": [
                {
                    "name": "Air Conditioner Advanced",
                    "type": "air-conditioner",
....
```

**Comment by jeylites** [comment]
I'm using a RM3 mini, I'm still waiting on the RM4 with temp dongle to come in the mail. I was switching between two RM3's troubleshooting the issue so I may have copied the wrong log. Honest mistake :) 

 Anyway I have corrected it and I'm still getting No Response on Homekit running on plugin version 4.2.9. I'm not sure how to go back to 4.2.8 but it was working before.




```
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 734-47-253 │     
    └────────────┘     
                       
[9/2/2020, 11:27:56] Homebridge is running on port 51155.
[9/2/2020, 11:27:56] [Broadlink RM] Air Conditioner Advanced getCurrentHeatingCoolingState: 2
[9/2/2020, 11:27:56] [Broadlink RM] Air Conditioner Advanced getTargetTemperature: 18
[9/2/2020, 11:27:56] [Broadlink RM] Air Conditioner Advanced getTargetHeatingCoolingState: 2
[9/2/2020, 11:27:56] [Broadlink RM] Fan Test getSwitchState: false
[9/2/2020, 11:27:56] [Broadlink RM] Fan Test getFanSpeed: 0
[9/2/2020, 11:27:56] [Broadlink RM] Fan Test getRotationDirection: 0
[9/2/2020, 11:27:56] [Broadlink RM] TV On/Off Test getSwitchState: false
[9/2/2020, 11:27:57] [Broadlink RM] [INFO] Discovered Broadlink RM Mini (2737) at 192.168.1.114 (#########)
[9/2/2020, 11:27:57] [Broadlink RM] Air Conditioner Advanced monitorTemperature
[9/2/2020, 11:27:57] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:27:58] [Broadlink RM] 
[9/2/2020, 11:27:58] [Broadlink RM] **************************************************************************************************************
[9/2/2020, 11:27:58] [Broadlink RM] ** Welcome to version 4.2.9 of the Homebridge Broadlink RM Plugin!
[9/2/2020, 11:27:58] [Broadlink RM] ** 
[9/2/2020, 11:27:58] [Broadlink RM] ** Find out what's in the latest release here: https://github.com/kiwi-cam/homebridge-broadlink-rm/blob/master/CHANGELOG.md
[9/2/2020, 11:27:58] [Broadlink RM] ** 
[9/2/2020, 11:27:58] [Broadlink RM] ** If you like this plugin then please star it on GitHub or better yet
[9/2/2020, 11:27:58] [Broadlink RM] ** buy me a drink using Paypal https://paypal.me/kiwicamRM.
[9/2/2020, 11:27:58] [Broadlink RM] **
[9/2/2020, 11:27:58] [Broadlink RM] ** You can disable this message by adding "hideWelcomeMessage": true to the config (see config-sample.json).
[9/2/2020, 11:27:58] [Broadlink RM] **
[9/2/2020, 11:27:58] [Broadlink RM] **************************************************************************************************************
[9/2/2020, 11:27:58] [Broadlink RM] 
[9/2/2020, 11:28:17] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:28:37] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:28:45] [Broadlink RM] Fan Test getSwitchState: false
[9/2/2020, 11:28:45] [Broadlink RM] Switch Test getSwitchState: undefined
[9/2/2020, 11:28:45] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:28:45] [Broadlink RM] Air Conditioner Advanced getTargetHeatingCoolingState: 2
[9/2/2020, 11:28:45] [Broadlink RM] Air Conditioner Advanced getTargetTemperature: 18
[9/2/2020, 11:28:45] [Broadlink RM] Air Conditioner Advanced getCurrentHeatingCoolingState: 2
[9/2/2020, 11:28:45] [Broadlink RM] Learn getSwitchState: undefined
[9/2/2020, 11:28:45] [Broadlink RM] TV On/Off Test getSwitchState: false
[9/2/2020, 11:28:45] [Broadlink RM] Scan Frequency getSwitchState: undefined
[9/2/2020, 11:28:57] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:29:17] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:29:37] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
[9/2/2020, 11:29:57] [Broadlink RM] Air Conditioner Advanced addTemperatureCallbackToQueue (requested temperature from device, waiting)
```

**Comment by kiwi-cam** [comment]
I know what has happened. A while a go I found an error in my code which mean Homebridge wouldn't poll the device for a temperature. This means that you could get a way with having no temperature source... which could give unexpected results.

Since your Broadlink doesn't have a thermometer you need to supply a temperature source. If you add either of these to your config, if should work:

key | description | example | default
-- | -- | -- | --
w1DeviceID | Updates device current temperature from a Raspberry Pi Wire-1 thermometers (i.e. ds18b20). Value is the Device ID | 28-0321544e531ff |  
pseudoDeviceTemperature | Some RM devices don't have a built in thermometer, when set this  prevents the device thermometer from being accessed and shows the  provided value instead. | 15
temperatureFilePath | A path to a file containing a number that represents the current temperature reported to the accessory. | "/Users/ username/ HomeKit/ temperature/"

**Comment by jeylites** [comment]
Made the changes and it It worked !!! 

I read about another user getting [MQTT Temp](https://github.com/lprhodes/homebridge-broadlink-rm/issues/482) to work with the plugin. I'm going to experiment with it shortly.

Does that mean to exclude `pseudoDeviceTemperature ,  w1DeviceID` from config?

Last but not least, I'm trying to get the Temp values to work correctly in Homekit. My AC  Low is 16C & High is 30C. There is No heater mode only cool. Is there a work around to get the correct temp to show up on Homekit app?

I've been experimenting setting the low to -9 and max to 0, as copied below. Though it now shows the correct value however, I'm not sure if this is the right way to do it.... 

THANK YOU!!!

```

"minTemperature": -9,
 "maxTemperature": 0,
              
```


** Config changes.** 
```
   "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "accessories": [
                {
                    "name": "Air Conditioner Advanced",
                    "type": "air-conditioner",
                    "allowResend": true,
                    "preventResendHex": true,
                    "ignoreTemperatureWhenOff": true,
                    "replaceAutoMode": "cool",
                    "autoSwitch": "A/C Auto Switch",
                    "hideWelcomeMessage": true,
                    "pseudoDeviceTemperature": 15,
                    "maxTemperature": 24,
                    "temperatureUpdateFrequency": 20,
                    "defaultCoolTemperature": 13,
                    "defaultHeatlTemperature": 24,
                    "data": {
                        "off": "26008a0093530b0d0b200b200d0c0b1f0c0c0c0d0b0d0b1f0c0c0c0c0c0c0b0d0b0d0b0d0d1e0d0c0b0c0c0c0c0d0b0d0b0d0b0c0c0c0c0c0b0d0b0d0b200d1e0d0c0c0c0c0c0c0c0c0c0c0c0c0c0b0d0b0e0a0d0d0c0b0c0c0d0b0c0c0d0b0c0c0c0c0c0c0c0c1f0b200b0d0b0d0d0c0c1e0c0c0b0d0b200d0c0b0c0c1f0c1f0c1f0c1f0c0c0c00029095000d050000000000000000000000000000",
                        "temperature10": {
                            "pseudo-mode": "cool",
                            "data": "0000 007A 0000 0043 00A0 005A 000D 000D 000D 0021 000D 0021 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 02AE 00A0 01E1"
                        },
```

**Comment by jeylites** [comment]
@kiwi-cam 

I realise there is another issue, I'm keep getting "Air Conditioner Advanced No cool HEX code found for 12" when there's a hex.  Then a default temp is used. 



```
***************
[9/3/2020, 00:57:51] [Broadlink RM] ** Welcome to version 4.2.9 of the Homebridge Broadlink RM Plugin!
[9/3/2020, 00:57:51] [Broadlink RM] ** 
[9/3/2020, 00:57:51] [Broadlink RM] ** Find out what's in the latest release here: https://github.com/kiwi-cam/homebridge-broadlink-rm/blob/master/CHANGELOG.md
[9/3/2020, 00:57:51] [Broadlink RM] ** 
[9/3/2020, 00:57:51] [Broadlink RM] ** If you like this plugin then please star it on GitHub or better yet
[9/3/2020, 00:57:51] [Broadlink RM] ** buy me a drink using Paypal https://paypal.me/kiwicamRM.
[9/3/2020, 00:57:51] [Broadlink RM] **
[9/3/2020, 00:57:51] [Broadlink RM] ** You can disable this message by adding "hideWelcomeMessage": true to the config (see config-sample.json).
[9/3/2020, 00:57:51] [Broadlink RM] **
[9/3/2020, 00:57:51] [Broadlink RM] **************************************************************************************************************
[9/3/2020, 00:57:51] [Broadlink RM] 
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced setTargetTemperature: 12
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced No cool HEX code found for 12
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced Update to default temperature (16)
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced sendHex (192.168.1.114; "$$:20:$$:1C:$$:$$") 26008a009a560c0c0c1f0c1f0c0c0c1f0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c1f0c1f0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c1f0c1f0c0c0c1f0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0002969a0001d00d05000000000000000000000000
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced getTargetTemperature: 16
[9/3/2020, 00:58:07] [Broadlink RM] Air Conditioner Advanced sentTemperature (16)
Command Acknowledged
[9/3/2020, 00:58:08] [Broadlink RM] Air Conditioner Advanced setCurrentHeatingCoolingState: 2
[9/3/2020, 00:58:20] [Broadlink RM] [ERROR] Air Conditioner Advanced No MQTT value could be found for topic with identifier "temperature".
[9/3/2020, 00:58:20] [Broadlink RM] Air Conditioner Advanced onTemperature (0)
[9/3/2020, 00:58:20] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/3/2020, 00:58:34] [Broadlink RM] Air Conditioner Advanced onTemperature (25.6)
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced setTargetTemperature: 11
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced No cool HEX code found for 11
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced Update to default temperature (16)
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced sendHex (192.168.1.114; "$$:20:$$:1C:$$:$$") 26008a009a560c0c0c1f0c1f0c0c0c1f0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c1f0c1f0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c0c1f0c1f0c0c0c1f0c0c0c0c0c0c0c1f0c0c0c0c0c0c0c0c0c1f0c0c0c0c0c0002969a0001d00d05000000000000000000000000
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced getTargetTemperature: 16
[9/3/2020, 00:58:44] [Broadlink RM] Air Conditioner Advanced sentTemperature (16)
[9/3/2020, 00:58:45] [Broadlink RM] Air Conditioner Advanced setCurrentHeatingCoolingState: 2
Command Acknowledged
[9/3/2020, 00:58:50] [Broadlink RM] Air Conditioner Advanced onTemperature (25.6)

```


**CONFIG**
```
{
    "bridge": {
        "name": "Homebridge 8483",
        "username": "$$:20:$$:1C:$$:$$",
        "port": 51155,
        "pin": "734-47-253"
    },
    "hosts": [
        {
            "address": "192.168.1.114",
            "mac": "$$:20:$$:1C:$$:$$",
            "isRFSupported": true
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "accessories": [
                {
                    "host": "192.168.1.114",
                    "name": "Air Conditioner Advanced",
                    "type": "air-conditioner",
                    "allowResend": true,
                    "preventResendHex": true,
                    "ignoreTemperatureWhenOff": true,
                    "replaceAutoMode": "cool",
                    "autoSwitch": "A/C Auto Switch",
                    "hideWelcomeMessage": true,
                    "mqttURL": "mqtt://192.168.1.200",
                    "mqttTopic": "tele/jeytemp/SENSOR",
                    "minTemperature": 10,
                    "maxTemperature": 30,
                    "temperatureUpdateFrequency": 30,
                    "defaultCoolTemperature": 16,
                    "data": {
                        "off": "26008a0093530b0d0b200b200d0c0b1f0c0c0c0d0b0d0b1f0c0c0c0c0c0c0b0d0b0d0b0d0d1e0d0c0b0c0c0c0c0d0b0d0b0d0b0c0c0c0c0c0b0d0b0d0b200d1e0d0c0c0c0c0c0c0c0c0c0c0c0c0c0b0d0b0e0a0d0d0c0b0c0c0d0b0c0c0d0b0c0c0c0c0c0c0c0c1f0b200b0d0b0d0d0c0c1e0c0c0b0d0b200d0c0b0c0c1f0c1f0c1f0c1f0c0c0c00029095000d050000000000000000000000000000",
                        "temperature16": {
                            "pseudo-mode": "cool",
                            "data": "0000 007A 0000 0043 00A0 005A 000D 000D 000D 0021 000D 0021 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 02AE 00A0 01E1"
                        },
                        "temperature17": {
                            "pseudo-mode": "cool",
                            "data": "0000 004A 0000 0043 0102 0090 0015 0015 0015 0035 0015 0035 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0035 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0454 0102 0307"
                        },
                        "temperature18": {
                            "pseudo-mode": "cool",
                            "data": "0000 006D 0000 0043 00B3 0064 000F 000F 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 02FD 00B3 0219"
                        },
                        "temperature19": {
                            "pseudo-mode": "cool",
                            "data": "0000 0070 0000 0043 00AC 0060 000E 000E 000E 0023 000E 0023 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 0023 000E 0023 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 0023 000E 000E 000E 02E0 00AC 0205"
                        },
                        "temperature20": {
                            "pseudo-mode": "cool",
                            "data": "0000 0055 0000 0043 00E8 0082 0013 0013 0013 0030 0013 0030 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0030 0013 0030 0013 03E1 00E8 02BA"
                        },
                        "temperature21": {
                            "pseudo-mode": "cool",
                            "data": "0000 0050 0000 0043 00F5 0089 0014 0014 0014 0032 0014 0032 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0032 0014 0032 0014 041A 00F5 02DE"
                        },
                        "temperature22": {
                            "pseudo-mode": "cool",
                            "data": "0000 0055 0000 0043 00E8 0082 0013 0013 0013 0030 0013 0030 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0030 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0030 0013 0013 0013 0013 0013 0013 0013 0030 0013 0030 0013 0030 0013 0030 0013 03E5 00E8 02BA"
                        },
                        "temperature23": {
                            "pseudo-mode": "cool",
                            "data": "0000 004E 0000 0043 00F5 0089 0014 0014 0014 0032 0014 0032 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0032 0014 0032 0014 0014 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0032 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 0014 041B 00F5 02DE"
                        },
                        "temperature24": {
                            "pseudo-mode": "cool",
                            "data": "0000 005A 0000 0043 00D3 0076 0011 0011 0011 002B 0011 002B 0011 0011 0011 002B 0011 0011 0011 0011 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 002B 0011 002B 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 0011 0011 002B 0011 0011 0011 0011 0011 0011 0011 0388 00D3 027A"
                        },
                        "temperature25": {
                            "pseudo-mode": "cool",
                            "data": "0000 004A 0000 0043 0102 0090 0015 0015 0015 0035 0015 0035 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0035 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0454 0102 0307"
                        },
                        "temperature26": {
                            "pseudo-mode": "cool",
                            "data": "0000 004B 0000 0043 0102 0090 0015 0015 0015 0035 0015 0035 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0035 0015 0015 0015 0015 0015 0015 0015 0035 0015 0035 0015 0015 0015 0015 0015 0453 0102 0307"
                        },
                        "temperature27": {
                            "pseudo-mode": "cool",
                            "data": "0000 006D 0000 0043 00B3 0064 000F 000F 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 0025 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 02FD 00B3 0219"
                        },
                        "temperature28": {
                            "pseudo-mode": "cool",
                            "data": "0000 0070 0000 0043 00AC 0060 000E 000E 000E 0023 000E 0023 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 0023 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 0023 000E 000E 000E 000E 000E 0023 000E 000E 000E 000E 000E 000E 000E 0023 000E 000E 000E 0023 000E 000E 000E 02E1 00AC 0205"
                        },
                        "temperature29": {
                            "pseudo-mode": "cool",
                            "data": "0000 007E 0000 0044 009B 0057 000D 000D 000D 0020 000D 0020 000D 000D 000D 0020 000D 000D 000D 000D 000D 000D 000D 000D 000D 0020 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0020 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0020 000D 0020 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0020 000D 000D 000D 000D 000D 0020 000D 000D 000D 0020 000D 000D 000D 000D 000D 0020 000D 000D 000D 000D 000D 000D 000D 000D 000D 0020 000D 0020 000D 000D 000D 0297 009B 0A9F 0007 01D1"
                        },
                        "temperature30": {
                            "pseudo-mode": "cool",
                            "data": "0000 007A 0000 0043 00A0 005A 000D 000D 000D 0021 000D 0021 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 000D 000D 000D 000D 0021 000D 000D 000D 000D 000D 000D 000D 000D 000D 0021 000D 0021 000D 0021 000D 02B0 00A0 01E1"
                        }
                    }
                },
                {
                    "name": "Switch Test",
                    "type": "switch"
                },
                {
                    "name": "Fan Test",
                    "type": "fan",
                    "data": {
                        "on": "2600500000012...",
                        "off": "2600500000012...",
                        "swingToggle": "2600500000012...",
                        "fanSpeed10": "2600500000012...",
                        "fanSpeed20": "2600500000012...",
                        "fanSpeed30": "2600500000012...",
                        "fanSpeed40": "2600500000012...",
                        "fanSpeed50": "2600500000012...",
                        "fanSpeed60": "2600500000012...",
                        "fanSpeed70": "2600500000012...",
                        "fanSpeed80": "2600500000012...",
                        "fanSpeed90": "2600500000012...",
                        "fanSpeed100": "2600500000012..."
                    }
                },
                {
                    "name": "TV On/Off Test",
                    "type": "switch",
                    "data": {
                        "on": "0000 006D 0000 0043 00B3 0064 000F 000F 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 0300 00B3 0219",
                        "off": "0000 006D 0000 0043 00B3 0064 000F 000F 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 0025 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 0025 000F 000F 000F 000F 000F 000F 000F 0025 000F 0025 000F 000F 000F 000F 000F 0300 00B3 0219"
                    }
                }
            ]
        }
    ],
    "accessories": [
        {
            "type": "motionSensor",
            "name": "CAT MOTION",
            "url": "192.168.1.200",
            "mqttPubOptions": {
                "retain": false
            },
            "topics": {
                "getMotionDetected": "stat/catlight/PIR1",
                "getStatusActive": "tele/catlight/STATE"
            },
            "onValue": "ON",
            "offValue": "OFF",
            "accessory": "mqttthing"
        }
    ]
}
```

**Comment by kiwi-cam** [comment]
You're right MQTT is the other option. You need to add **EITHER** of these to stop it checking the Broadlink for the temperature:
key | description | example | default
-- | -- | -- | --
w1DeviceID | Updates device current temperature from a Raspberry Pi Wire-1 thermometers (i.e. ds18b20). Value is the Device ID | 28-0321544e531ff |  
pseudoDeviceTemperature | Some RM devices don't have a built in thermometer, when set this  prevents the device thermometer from being accessed and shows the  provided value instead. | 15
temperatureFilePath | A path to a file containing a number that represents the current temperature reported to the accessory. | "/Users/ username/ HomeKit/ temperature/"
mqttURL | The URL to retrieve the temperature | mqtt://192.168.0.199

If you go the MQTT route, you'll also need to add mqttTopic and optionally, mqttUsername and mqttPassword (Reference: https://lprhodes.github.io/slate/#air-conditioner).

**Comment by jeylites** [comment]
Sorry I did't respond back. I have tested the MQTT temp configuration and it works. Only sometimes I get an Error, sometimes the problem goes away after I do a ESP reset. 

                       
```
[9/21/2020, 12:25:34] Homebridge v1.2.3 is running on port 51155.
[9/21/2020, 12:25:34] [Broadlink RM] JEY AC getCurrentHeatingCoolingState: 2
[9/21/2020, 12:25:34] [Broadlink RM] JEY AC getTargetTemperature: 18
[9/21/2020, 12:25:34] [Broadlink RM] JEY AC getTargetHeatingCoolingState: 2
[9/21/2020, 12:25:34] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:25:34] [Broadlink RM] Switch Test getSwitchState: false
[9/21/2020, 12:25:34] [Broadlink RM] [INFO] JEY AC MQTT client connected.
[9/21/2020, 12:25:34] [Broadlink RM] Fan Test getSwitchState: false
[9/21/2020, 12:25:34] [Broadlink RM] Fan Test getFanSpeed: 29
[9/21/2020, 12:25:34] [Broadlink RM] Fan Test getRotationDirection: 0
[9/21/2020, 12:25:34] [Broadlink RM] TV On/Off Test getSwitchState: false
[9/21/2020, 12:25:34] [Broadlink RM] [INFO] Discovered Broadlink RM Mini (2737) at 192.168.1.105 (##########)
[9/21/2020, 12:25:35] [Broadlink RM] JEY AC monitorTemperature
[9/21/2020, 12:25:35] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:25:35] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:25:35] [Broadlink RM] 
[9/21/2020, 12:25:35] [Broadlink RM] **************************************************************************************************************
[9/21/2020, 12:25:35] [Broadlink RM] ** Welcome to version 4.0.5 of the Homebridge Broadlink RM Plugin!
[9/21/2020, 12:25:35] [Broadlink RM] ** 
[9/21/2020, 12:25:35] [Broadlink RM] ** Find out what's in the latest release here: https://github.com/lprhodes/homebridge-broadlink-rm/releases
[9/21/2020, 12:25:35] [Broadlink RM] ** 
[9/21/2020, 12:25:35] [Broadlink RM] ** If you like this plugin then please star it on GitHub or better yet
[9/21/2020, 12:25:35] [Broadlink RM] ** buy me a drink using Paypal https://paypal.me/lprhodes or crypto https://goo.gl/bEn1RW.
[9/21/2020, 12:25:35] [Broadlink RM] ** 
[9/21/2020, 12:25:35] [Broadlink RM] ** Keep up to date with this plugin along with everything HomeKit and homebridge
[9/21/2020, 12:25:35] [Broadlink RM] ** by signing up to my newsletter at http://workswith.io
[9/21/2020, 12:25:35] [Broadlink RM] **
[9/21/2020, 12:25:35] [Broadlink RM] ** You can disable this message by adding "hideWelcomeMessage": true to the config (see config-sample.json).
[9/21/2020, 12:25:35] [Broadlink RM] **
[9/21/2020, 12:25:35] [Broadlink RM] **************************************************************************************************************
[9/21/2020, 12:25:35] [Broadlink RM] 
[9/21/2020, 12:25:45] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:25:45] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:25:55] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:25:55] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:26:05] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:26:05] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:26:15] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:26:15] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:26:25] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
[9/21/2020, 12:26:25] [Broadlink RM] [INFO] Reported temperature (0) is too low, setting to minTemperature (10).
[9/21/2020, 12:26:35] [Broadlink RM] [ERROR] JEY AC No MQTT value could be found for topic with identifier "temperature".
```

**Comment by jeylites** [comment]
@kiwi-cam I tried to contribute to your Paypal using the link provided in the logs but it's broken.

**Comment by kiwi-cam** [comment]
That’s a severity 1 issue 😊

The one on the Readme works: https://paypal.me/kiwicamRM

**Comment by kiwi-cam** [comment]
> @kiwi-cam I tried to contribute to your Paypal using the link provided in the logs but it's broken.

@jeylites Do you mean this link:
```
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **************************************************************************************************************
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] ** Welcome to version 4.3.3-beta.1 of the Homebridge Broadlink RM Plugin!
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] ** Find out what's in the latest release here: https://github.com/kiwi-cam/homebridge-broadlink-rm/blob/master/CHANGELOG.md
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] ** If you like this plugin then please star it on GitHub or better yet
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] ** buy me a drink using Paypal https://paypal.me/kiwicamRM.
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] ** You can disable this message by adding "hideWelcomeMessage": true to the config (see config-sample.json).
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **
Sep 22 08:45:16 homebridge[28742]: [Broadlink RM] **************************************************************************************************************
```

I can't find an incorrect link. Could you share the full text you're seeing?

**Comment by jeylites** [comment]
@kiwi-cam 

I found the issue, if you click on the link on the log, it includes the full stop symbol after the .... [/kiwicamRM. ](https://paypal.me/kiwicamRM.)

Anyway, I just Paypal you.
